providers/oidc_test.go

```diff
@@ -167,6 +167,14 @@ func TestOIDCProviderRedeem(t *testing.T) {
 	assert.Equal(t, "123456789", session.User)
 }
 
+func TestOIDCProviderGetOidcInfoToken(t *testing.T) {
+	body, _ := json.Marshal(defaultIDToken)
+	server, provider := newTestSetup(body)
+	defer server.Close()
+	_, err := provider.GetOidcInfoToken(context.Background(), accessToken)
+	assert.Equal(t, nil, err, "Error while Accessing oidc token")
+}
+
```

Comment:
Can we extend the tests please, would be good to see what a successful response is supposed to look like